modules/sync/state.json

```diff
@@ -69,4 +69,4 @@
       "latest_reference": "v25.1"
     }
   ]
-}
+}
```

Comment:
thought it was weird that this changed now, but we manually changed it in the previous commit https://github.com/bufbuild/modules/commit/782ccc6b09f09484f72c0429b0d81303e1d98e17#diff-66b88475781fe3747cdd841d516aee1dcc60c8ab2e93f00a26c89a9a15b0451c :-)

Reply:
Yeah that was me 😅 sorry